### ISSUE TITLE
First Card does not render when Second Card gets replaced with null

### ISSUE THREAD

**Issue by TikiTDO** [alexbrillant/react-native-deck-swiper#10]
I hit a really, really strange problem while using this lib with RN 0.44.0 on an android 7.1.2 device.

When I have a stack of 2 or more cards, and I swipe to the last card in the stack, the final card does not appear, and the onSwipedAll event does not fire. 

I was able to isolate the problem down to [this line](https://github.com/alexbrillant/react-native-deck-swiper/blob/master/Swiper.js#L358) in the `renderSecondCard` method. When the Swiper is re-rendered after going from a state where `renderSecondCard` returns an RN element, to a state where `renderSecondCard` returns a null, the first card is not rendered.

I was able to fix the issue by changing the following code:

```js
    // This broke
    if (notInfinite && lastCardOrSwipedAllCards) {
      return null
    }
```
To: 

```js
    // This worked correctly
    if (notInfinite && lastCardOrSwipedAllCards) {
      return (
        <Animated.View />
      )
    }
```

However, it appears that it has to be Animated.View. When I tried substituting a normal View, it broke just like with the null case:

```js
    // This also broke
    if (notInfinite && lastCardOrSwipedAllCards) {
      return (
        <View />
      )
    }
```

My gut feeling is that this might be an issue with how RN tries to re-use or GCs components, but I'm far too tired to investigate further for now. I would appreciate if you could look into this further, or if you could release a version with the working fix from above. If neither of those are acceptable then I should have some time to look at it in more detail sometime over the weekend.

**Comment by jpaas** [comment]
When I moved up to 1.1.5 I started having problems with android. I have a deck with 4 cards. When I swipe card 3, I can see that card 3 & 4 are animated, but the logging shows that card 3 was swiped, so I'm left with just my background and I can't see card 4, so I end up stuck.

**Comment by alexbrillant** [comment]
Thanks for sharing. I will fix it as soon as possible.

**Comment by alexbrillant** [comment]
@TikiTDO I pushed your fix into version 1.1.6 !
